lib/vue2-version-warn.js

```diff
@@ -0,0 +1,10 @@
+var chalk = require('chalk')
+
+module.exports = function (template, name) {
+  var vue1InitCommand = 'vue init ' + template + '#1.0' + ' ' + name
```

Comment:
Should we also pass in `inPlace` and only add the `name` to this string if it's false?